src/app/static/src/app/components/baseline/Baseline.vue

```diff
@@ -65,16 +68,19 @@
                 pjnz: state => ({
                     valid: !!state.country,
                     error: state.pjnzError,
+                    adrValid: state.pjnz?.fromADR,
                     existingFileName: state.pjnz && state.pjnz.filename
                 } as PartialFileUploadProps),
                 shape: state => ({
                     valid: state.shape != null,
                     error: state.shapeError,
+                    adrValid: state.shape?.fromADR,
```

Comment:
This prop name doesn't quite make sense - the flag is just showing whether files comes from ADR vs from the user's local computer, has nothing to do with validity, so can we change `adrValid` to `fromADR`?